fabric-datafixer-v0/src/main/java/net/fabricmc/fabric/api/datafixer/SimpleFixes.java

```diff
@@ -0,0 +1,96 @@
+package net.fabricmc.fabric.api.datafixer;
+
+import com.google.common.collect.ImmutableMap;
+import com.mojang.datafixers.DataFixerBuilder;
+import com.mojang.datafixers.Dynamic;
+import com.mojang.datafixers.schemas.Schema;
+import com.mojang.datafixers.util.Pair;
+
+import net.fabricmc.fabric.impl.datafixer.FabricSimpleFixesImpl;
+import net.minecraft.datafixers.fixes.EntitySimpleTransformFix;
+
+public abstract class SimpleFixes {
+    
+    private static SimpleFixes INSTANCE = FabricSimpleFixesImpl.INSTANCE;
+    
+    /**
+     * A basic skeleton DataFix for changing block names.
+     * @param builder_1 The builder to add this fix to.
+     * @param name The name of the datafix (this has no effect on actual process)
+     * @param changes A map of all changed values, where the key should be original and value is new value.
+     * @param schema_1 The Schema to apply this fix to.
+     */
+    public static void addBlockRenameFix(DataFixerBuilder builder_1, String name, ImmutableMap<String, String> changes, Schema schema_1) {
```

Comment:
Drop the _1 from the param names. just `builder` and `schema`. I guess you got the idea for these from the decompiler.